pkg/auth/auth.go

```diff
@@ -24,10 +24,12 @@ import (
 	"path"
 	"strings"
 
-	logging "github.com/op/go-logging"
 	"github.com/openshift/ansible-service-broker/pkg/config"
+	utillogging "github.com/openshift/ansible-service-broker/pkg/util/logging"
 )
 
+var log = utillogging.NewLog()
```

Comment:
Ah so each package does this call. I was searching for where the hell the logger was set. Found it.